README.md

```diff
@@ -1,3 +1,80 @@
-# The Nivenly Covenant
+# The Nivenly Covenant 1.0
+
+Include a published and publicly accessible statement you to agree and commit to the following terms. 
+
+``` 
+<Entity,Person(s),Authors,Organization(s)> herby agree an unequivocally commit to the fullest
+extent of our understanding, our intentions, and our goals to The Nivenly Covenant 1.0
+```
+
+### 1. Sustainability
+
+We agree that sustainability is our critical priority, and is necessary for the longevity of future generations of stewards and technologists.
+
+We agree to prioritize the sustainability, mental, and physical health, well-being, overall working condition, and operational excellence for the people working on the project, the source code of the project, and the day-to-day operations of the project.
+
+We understand that the longevity and success of this project is intended to be given to another generation of stewards long after the original authors have completed their work on the project. We aim to leave this project in better condition than we inherited it.
+
+We understand that "right" is more important than "right now", and that future generations are more important than the needs of today.
+
+Given the decision between profit and sustainability, we unequivocally chose sustainability.
```

Comment:
We need to have some phrasing around "We understand the difference between not making a profit right now and actively running at a deficit." Basically I want to have something that prompts active thought that "not at a profit" doesn't immediately watersled into "hemorrhaging money".

Reply:
i _think_ this is part of the definition of sustainability but it would be good to call that out.